flink-connectors/flink-avro/src/main/java/org/apache/flink/api/java/io/AvroOutputFormat.java

```diff
@@ -125,9 +125,7 @@ public void open(int taskNumber, int numTasks) throws IOException {
 			datumWriter = new SpecificDatumWriter<E>(avroValueType);
 			try {
 				schema = ((org.apache.avro.specific.SpecificRecordBase)avroValueType.newInstance()).getSchema();
-			} catch (InstantiationException e) {
-				throw new RuntimeException(e.getMessage());
-			} catch (IllegalAccessException e) {
+			}catch(InstantiationException | IllegalAccessException  e) /*multi-catch refactor*/ {
```

Comment:
Why change the whitespace before 'catch' and why include the comment? There's an extra space after each `IllegalAccessException`.